doc/source/servers/batch.md

```diff
@@ -27,6 +27,7 @@ This allows you to leverage Seldon on a large number of batch applications, incl
 We have provided a set of examples that show you how you can use the Seldon batch processing component:
 
 * [Batch Processing with Argo Workflows](../examples/argo_workflows_batch.html)
```

Comment:
May be worth renaming "Argo Workflows and S3 / Minio"

Reply:
Done.